src/ILLink.Tasks/build/Microsoft.NET.ILLink.targets

```diff
@@ -225,16 +238,9 @@ Copyright (c) .NET Foundation. All rights reserved.
       <ILLinkWarningLevel Condition=" '$(ILLinkWarningLevel)' == '' ">0</ILLinkWarningLevel>
     </PropertyGroup>
 
-    <!-- Defaults for .NET < 6 -->
-    <PropertyGroup Condition=" $([MSBuild]::VersionLessThan('$(TargetFrameworkVersion)', '6.0')) ">
-      <TrimMode Condition=" '$(TrimMode)' == '' ">copyused</TrimMode>
-      <!-- Action is the same regardless of whether the assembly has an IsTrimmable attribute. (The attribute didn't exist until .NET 6.) -->
-      <TrimmerDefaultAction>$(TrimMode)</TrimmerDefaultAction>
-    </PropertyGroup>
     <PropertyGroup>
-      <TrimMode Condition=" '$(TrimMode)' == '' ">link</TrimMode>
-      <!-- For .NET 6+, assemblies without IsTrimmable attribute get the "copy" action. -->
-      <TrimmerDefaultAction Condition=" '$(TrimmerDefaultAction)' == '' ">copy</TrimmerDefaultAction>
+      <!-- Set TrimmerDefaultAction for compat -->
+      <_TrimmerDefaultAction>$(TrimmerDefaultAction)</_TrimmerDefaultAction>
```

Comment:
What's the reason to introduce a private version of this property - we don't seem to use it for anything really.